src/chapters/objects-and-math/modifyingobjects.rst

```diff
@@ -0,0 +1,145 @@
+Working with Objects
+====================
+
+Accessing Properties
+--------------------
+
+When using objects, programmers oftentimes want to retrieve or change the value of one of the properties.
+To access the value of a property, you will need the object's name and the key of the property.
+
+Programmers have two ways to access the value of property:
+
+1. Bracket syntax
+2. Dot notation.
+
+Bracket Syntax
+^^^^^^^^^^^^^^
+
+Using brackets, the code would be something like: ``object["key"]``. The only restraint in naming a key is that it has to be a valid JavaScript string.
+Since a key could potentially have a space in it, bracket syntax would be the only way to access the value in that property because of the quotes.
+
+Dot Notation
+^^^^^^^^^^^^
+
+With dot notation, the code would be something like: ``object.key``. Notice that the key is no longer surrounded by quotes. However, keys are still strings.
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+      :linenos:
+
+      let tortoiseOne = {
+          species: "Galapagos Tortoise",
+          name: "Pete",
+          weight: 919,
+          age: 85,
+          diet: ["pumpkins", "lettuce", "cabbage"]
+      };
+
+      console.log(tortoiseOne["name"]);
+      console.log(tortoiseOne.name);
+
+   **Console Output**
+
+   ::
+
+      Pete
+      Pete
+
+Modifying Properties
+--------------------
+
+A programmer can modify the value that a property holds by using either notation above.
```

Comment:
Rephrasing suggestion: How about, "A programmer can modify the value of a property by using either notation style."

Reply:
Done!